apps/linux/gdb.talon

```diff
@@ -0,0 +1,108 @@
+os: linux
+app: /term/
```

Comment:
maybe use tag: terminal instead of app?

Reply:
will do. also been playing around with a 'debug mode' that triggers `mode: gdb` to get around the script problem and some new problems from me switching to using vim as my terminal. so will add that too.

Reply:
Oki doki, I'll leave this open and wait for these tweaks.